Netskope improvements (#37493)
demisto/content: Packs/Netskope/Integrations/NetskopeEventCollector/README.md

## Fetch Events Limitation

> The collector can handle 35K events per minute on average per each event type  

Reviewer: but this is not true... I think this is better no?
```suggestion
The collector can handle up to 35K events per minute on average.
```
